### DISCUSSION TITLE
Uploading very large files (>25 Gb) in ckan 2.10

### DISCUSSION THREAD

**Discussion by ChristianF88** [ckan/ckan#7429] [General]
Hi everyone

I am managing a ckan instance for a scientific institute. Data that is uploaded into our repository ranges from a couple of Mb to 100 Gb in size. What's the recommended/best (reliable and fast) way to upload very large files (**>25 Gb**) into **ckan 2.10**? How do you approach this? 

Cheers

**Comment by anuveyatsu** [comment]
@ChristianF88 hi, the main point would be uploading directly to storage (eg, if you're using S3 or similar). Take a look here: https://github.com/datopian/ckanext-blob-storage
